git-scripts.gemspec

```diff
@@ -19,10 +19,9 @@ Gem::Specification.new do |s|
               'copperwall@gmail.com',
               'robin@ifixit.com']
 
-   s.add_dependency 'bundler'
-   s.add_dependency 'octokit', '~> 4.0.0'
-   s.add_dependency 'highline'
-   s.add_dependency 'json', '~> 1.8.0'
+   s.add_dependency 'bundler', '~> 1.17'
+   s.add_dependency 'octokit', '~> 4.0'
+   s.add_dependency 'json', '~> 1.8'
```

Comment:
I don't think we need this as an explicit dep any longer. We're running ruby > 2.4 everywhere these scripts get used...